pandajedi/jedicore/JediDBProxy.py

```diff
@@ -13784,8 +13785,22 @@ def push_task_trigger_message(self, msg_type, jedi_task_id, data_dict=None, prio
                 return
             if mb_proxy.got_disconnected:
                 mb_proxy.restart()
-            if priority:
-                mb_proxy.send(msg, priority=priority)
+            # message priority
+            msg_priority = None
+            if priority is not None:
+                msg_priority = priority
+            elif task_spec is not None:
+                try:
+                    if task_spec.prodSourceLabel == "user":
+                        if task_spec.gshare in ["User Analysis", "Express Analysis"]:
```

Comment:
It doesn't look nice to hardcode gshare names. Better to have a new attribute in ATLAS_PANDA.GLOBAL_SHARES?

Reply:
> new attribute in ATLAS_PANDA.GLOBAL_SHARES

Does it mean to add new columns in GLOBAL_SHARES tables?
It's reason if gshare is the only concern to define "MD-priority", but I'm not sure it will be the case. Will interactive tasks own a new gshare or tagged by other attributes?

Another thought: This part is just to pass the value of msg_priority for MQ. Now spiltrule only has the possibility MD=1, but maybe it is fine to have splitrule MD=2 (or 3 or 4 etc.) to represent different task priority for MD. While the part in other module (task refiner?) where to mark the MD splitrule may still be kinda hard-coded if mixed attributes are required to define priority of MD.